docs/user-guide/cli-usingcli.md

```diff
@@ -31,41 +32,331 @@ zowe --ac
+
+**Note:** To access API ML, ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the token in your base profile and directly access the service.
+
+#### Specifying base path
+
+<!--
+TODO - I'm a little iffy on this section. Are you specifying a base path to zosmf, or just the base path to APIML and it figures out which zosmf instance from there? Generally, is this section correct?
+-->
+
+The following example illustrates a complete path for a z/OSMF instance registered to API ML. The format of base path can vary based on how API ML is configured at your site:
+
+```
+https://myapilayerhost:port/api/v1/zosmf1
+```
+
+To access that API ML instance, create a service profile (or issue a command) with the `--base-path` value of `api/v1`. Your service profile references the session token in your base profile. If you have multiple base profles, you can specify one with the `--base-profile` option. For example:
+
+```
+zowe profiles create zosmf myprofile123 --base-path api/v1 --base-profile mybaseprofile123
+```
+
+Commands that you issue with this profile are routed through the layer to access an appropriate instance of z/OSMF.
+
+### Accessing multiple services with SSO
+
+If multiple services are registered to the API Mediation Layer at your site, Zowe CLI lets you access the services with Single Sign-on (SSO). Log in once to create a session and conveniently access all available services.
+
+When you are logged-in, supply the `--base-path` on commands to use the secure session for each service. Ensure that you do not provide username, password, host, or port directly on your service commands or profiles. Supplying those options causes the CLI to ignore the token in your base profile and directly access the service. You might need to remove those options from existing profiles to use SSO.
+
+For information about registering an API service at your site, see [Developing for API Mediation Layer](../extend/extend-apiml/onboard-overview.md).
+
+### Accessing multiple services through SSO + one service not through APIML
+
+There might be a scenario where you initiated an SSO session with API ML, but you also want access a different service directly (not through API ML).
+
+To access the SSO-enabled services, log in and issue commands with the `--base-path` and `--base-profile` options. The token from your base profile is used for authentication. Remember, your command or service profile must *not* contain username, password, host, or port.
+
+To access the other service directly and circumvent API ML, supply all connection information (username, password, host, and port) on a command or service profile. When you explicitly supply a username and password in a command or service profile, the CLI always uses that connection information instead of the API ML session token.
+
+<!-- TODO
```

Comment:
I think the use case is someone wants to use APIML + SSO wherever possible at their site, but also wants to use a service that doesn't support SSO. 

> My first guess is that you would do the usual SSO w/ base path and token for the first 2 services

That is correct.

> then for the non-SSO service you would specify base path, but also specify username, host, port, etc...?

You would specify base path, user, and password. Host and port aren't needed as they're identical to the values in the base profile, since the service is accessed through the same APIML instance that supports SSO.

> Would that cause it to go to APIMl, but then choose a specific service instance?

The base path will point to a specific service ID on APIML, which is the same behavior as for services that support SSO. The difference is that a user and password are supplied which are specific to the service, rather than a user/password or token that are applicable for all services (SSO).

Reply:
Can you please review my attempt at this description? Thx

Reply:
What I saw in the latest update looked correct to me.